### DISCUSSION TITLE
How to use libraries with `.(s)css` not `.module.(s)css` files?

### DISCUSSION THREAD

**Discussion by varya** [vercel/next.js#18838] [Help]
How do people use not compiled libraries with NextJs? I am linking the library which has `.scss` files for the components (imported in their `.js`) but NextJs gives an error because the file is not named as `.module.scss`. I haven’t found any workaround via config. Should the library be forced to rename the files?

**Comment by lfades** [comment]
`.module.css` works for components inside the Next.js app, for `node_modules` you should be able to import the css directly: https://nextjs.org/docs/basic-features/built-in-css-support#import-styles-from-node_modules, we also have [Sass support](https://nextjs.org/docs/basic-features/built-in-css-support#sass-support)

**Reply by varya** [reply]
Thank you, I read this. I don't import the CSS directly, I import js file from the library which is in `node_modules` (and it imports CSS) but I got an error that the file is not named as `.module.scss`.
